WordPress/src/main/java/org/wordpress/android/ui/posts/FeaturedImageHelper.kt

```diff
@@ -137,6 +149,24 @@ internal class FeaturedImageHelper @Inject constructor(
         return FeaturedImageData(FeaturedImageState.REMOTE_IMAGE_LOADING, photonUrl)
     }
 
+    fun trackFeaturedImageEvent(
+        stat: Stat,
+        post: PostImmutableModel?
```

Comment:
It feels a bit weird that the post is nullable. Featured image always requires a post. We are passing null just because we don't have an access to the post object from the `PhotoPickerActivity`.  I'd consider replacing `PostImmutableModel?` with a post id to which we have access even from the Activity (unless we know we need to track "post_status"). Wdyt?